motoko/icrc2-swap/README.md

```diff
@@ -0,0 +1,311 @@
+# ICRC-2 Swap
```

Comment:
Very well written! Thank you very much for putting in so much effort. I only have some minor comments